packages/provider/src/GSNConfigurator.ts

```diff
@@ -53,13 +55,6 @@ export const defaultGsnConfig: GSNConfig = {
   performDryRunViewRelayCall: true
 }
 
-export interface LoggerConfiguration {
```

Comment:
Not strictly related to this refactor. ..
Though I agree its not really needed.